lib/cocoapods/installer/target_installer/aggregate_target_installer.rb

```diff
@@ -113,9 +113,12 @@ def resources_by_config
         resources_by_config = {}
         target.user_build_configurations.keys.each do |config|
           file_accessors = library_targets.select { |t| t.include_in_build_config?(config) }.flat_map(&:file_accessors)
-          resource_paths = file_accessors.flat_map { |accessor| accessor.resources.flat_map { |res| res.relative_path_from(project.path.dirname) } }
-          resource_bundles = file_accessors.flat_map { |accessor| accessor.resource_bundles.keys.map { |name| "${BUILT_PRODUCTS_DIR}/#{name.shellescape}.bundle" } }
-          resources_by_config[config] = (resource_paths + resource_bundles).uniq
+          resource_bundles = file_accessors.flat_map do |accessor|
+            accessor.resource_bundles.keys.map do |name|
+              Pathname.new "${BUILT_PRODUCTS_DIR}/#{name.shellescape}.bundle"
```

Comment:
I actually didn't change the code here (I merely replaced `{}` with `do…end` to avoid a long line), so I don't know who is the author of that and why he did that.